src/styles/main.scss

```diff
@@ -1 +1,63 @@
-// put code here
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  padding-top: 100px;
+}
+
+.watch {
+  @import "vars.scss";
+
+  position: relative;
+  box-sizing: border-box;
+  height: $watch-size;
+  width: $watch-size;
+  border: 19px dotted #a35561;
+  outline: 3px groove #b86b77;
+  background-color: #f6cfca;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)